pkgs/applications/video/aegisub/default.nix

```diff
@@ -2,162 +2,214 @@
 , libpulseaudio ? null
-
 , portaudioSupport ? false
 , portaudio ? null
-
-, useBundledLuaJIT ? false
-, darwin
+,
 }:
-
 assert spellcheckSupport -> (hunspell != null);
 assert openalSupport -> (openal != null);
 assert alsaSupport -> (alsa-lib != null);
 assert pulseaudioSupport -> (libpulseaudio != null);
-assert portaudioSupport -> (portaudio != null);
+assert portaudioSupport -> (portaudio != null); let
+  inherit (lib) optional;
 
-let
   luajit52 = luajit.override { enable52Compat = true; };
-  inherit (lib) optional;
-  inherit (darwin.apple_sdk.frameworks) CoreText CoreFoundation AppKit Carbon IOKit Cocoa;
+
+  full_version = "feature_11";
+
+  bestsource = fetchFromGitHub {
+    owner = "vapoursynth";
+    repo = "bestsource";
+    rev = "ba1249c1f5443be6d0ec2be32490af5bbc96bf99";
+    hash = "sha256-9BnyRzF33otju3W503O18JuTyvp+hFxk6JMwrozKoZY=";
+    fetchSubmodules = true;
+  };
+
+  vapoursynth = fetchFromGitHub {
+    owner = "vapoursynth";
+    repo = "vapoursynth";
+    rev = "R65";
+    hash = "sha256-6w7GSC5ZNIhLpulni4sKq0OvuxHlTJRilBFGH5PQW8U=";
+  };
+
+  AviSynthPlus = fetchFromGitHub {
+    owner = "AviSynth";
+    repo = "AviSynthPlus";
+    rev = "v3.7.3";
+    hash = "sha256-sBxZ2J5sS/2wrL+tSxVAFPKNbg3c1iMSHRueRQAs5J4=";
+    fetchSubmodules = true;
+  };
+
+  gtest = fetchurl {
+    url = "https://github.com/google/googletest/archive/release-1.8.1.zip";
+    hash = "sha256-kngnwYPQFzTMXP74Xg/z9akv/mGI4NGOkJxe/r8ooMc=";
+  };
+
+  gtest_patch = fetchurl {
+    url = "https://wrapdb.mesonbuild.com/v1/projects/gtest/1.8.1/1/get_zip";
+    hash = "sha256-959f1G4JUHs/LgmlHqbrIAIO/+VDM19a7lnzDMjRWAU=";
+  };
 in
 stdenv.mkDerivation rec {
   pname = "aegisub";
-  version = "3.3.3";
+  version = full_version;
 
   src = fetchFromGitHub {
-    owner = "wangqr";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-oKhLv81EFudrJaaJ2ga3pVh4W5Hd2YchpjsoYoqRm78=";
+    owner = "arch1t3cht";
+    repo = "Aegisub";
+    rev = full_version;
+    hash = "sha256-yEXDwne+wros0WjOwQbvMIXk0UXV5TOoV/72K12vi/c=";
   };
 
   nativeBuildInputs = [
-    intltool
-    luajit52
-    pkg-config
-    which
+    meson
+    ninja
     cmake
-    wrapGAppsHook
+    pkg-config
+    luajit52
+    gettext
+    python3
   ];
 
-  buildInputs = [
-    boost179
-    expat
-    ffmpeg
-    ffms
-    fftw
-    fontconfig
-    freetype
-    fribidi
-    glib
-    harfbuzz
-    icu
-    libGL
-    libGLU
-    libX11
-    libass
-    libiconv
-    libuchardet
-    pcre
-    wxGTK
-    zlib
-  ]
-  ++ lib.optionals stdenv.isDarwin [
-    CoreText
-    CoreFoundation
-    AppKit
-    Carbon
-    IOKit
-    Cocoa
-  ]
-  ++ optional alsaSupport alsa-lib
-  ++ optional openalSupport openal
-  ++ optional portaudioSupport portaudio
-  ++ optional pulseaudioSupport libpulseaudio
-  ++ optional spellcheckSupport hunspell
-  ;
-
-  enableParallelBuilding = true;
+  buildInputs =
+    [
+      fontconfig
```

Comment:
Please place the reformatting in a separate commit before the changes.

(Nor does it follow [RFC 166](https://github.com/NixOS/rfcs/blob/master/rfcs/0166-nix-formatting.md): do you use formatters other than `nixfmt-rfc-style`?)